docs/gl_objects/merge_requests.rst

```diff
@@ -189,6 +189,12 @@ Attempt to rebase an MR::
 
     mr.rebase()
 
+
+Attempt to reset all MR approval::
```

Comment:
Same as below, and maybe we could mention this only works for bots to avoid questions from users in the future, WDYT?

```suggestion
Clear all approvals of a merge request (possible with project or group access tokens only)::
```

Reply:
Added.